tests/st/bgp/test_ipip.py

```diff
@@ -287,3 +287,51 @@ def get_tunl_tx(self, host):
         match = re.search(r'RX packets:(\d+) ',
                           output)
         return int(match.group(1))
+
+    def test_ipip_gce(self):
+        """
+        Test IPIP routing on simulated GCE instances.
+        """
+        with DockerHost('host1',
+                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
+                        simulate_gce_routing=True,
+                        start_calico=False) as host1, \
+             DockerHost('host2',
+                        additional_docker_options=CLUSTER_STORE_DOCKER_OPTIONS,
+                        simulate_gce_routing=True,
+                        start_calico=False) as host2:
+
+            host1.start_calico_node()
+            host2.start_calico_node()
+
```

Comment:
It occurs to me that we should also be testing a failure case when IPIP is not enabled - otherwise we don't have any visibility that the emulated GCE network is really doing its thing.  WDYT?